### DISCUSSION TITLE
Eclipse CDT product branding

### DISCUSSION THREAD

**Discussion by hparpinel** [eclipse-cdt/cdt#373] [Q&A]
Hi,

I am trying to customise my Eclipse CDT installation with my own icon, name, and environment layout. I am new to this and I am not sure if I should add my plugins to my existing install or if I should start here https://github.com/eclipse-embed-cdt/eclipse-plugins. Is there a more extended guide somewhere?

Thanks
H

**Comment by ruspl-afed** [comment]
Hi,

It could be better to start from here https://help.eclipse.org/latest/topic/org.eclipse.platform.doc.isv/guide/product_configproduct.htm?cp=2_0_21_0_1
